### ISSUE TITLE
Single dollar sign stops interpolation

### ISSUE THREAD

**Issue by koterpillar** [tmhedberg/here#19]
```
λ> :set -XQuasiQuotes 
λ> import Data.String.Here
λ> putStrLn [i|${"one"} $ ${"two"}|]
one $ ${"two"}
```

**Comment by 3noch** [comment]
Aha! So *that's* what made the parser fail for me. +1

**Comment by domenkozar** [comment]
I got bitten by the same thing (sadly dollars are all over the place in Nix).

```
λ> let foo = "haha"                                                                  │·
λ> [here|so $foo and ${foo}|]                                                        │·
"so $foo and ${foo}"            
```

For now we could at least fix documentation to note that `$` has to be escaped additional to `${`.

**Comment by tmhedberg** [comment]
I think I've fixed this in 8a616b3. It's on the `master` branch. Please try it out with any real inputs that you have and let me know if you still encounter any problems.

**Comment by domenkozar** [comment]
@tmhedberg confirmed that https://github.com/domenkozar/elm2nix/commit/13455d941bf5fe90073d48d52428ae7763fbe85e works for my use case.

**Comment by tmhedberg** [comment]
I've just published this fix in [v1.2.12](http://hackage.haskell.org/package/here-1.2.12).
